src/shared/schemas/balance-subscription-config.ts

```diff
@@ -0,0 +1,33 @@
+import { Type } from "@sinclair/typebox";
+import { z } from "zod";
+
+// TypeBox schema for request validation
+export const balanceSubscriptionConfigSchema = Type.Object({
+  threshold: Type.Optional(
+    Type.Object({
+      min: Type.Optional(Type.String({
+        description: "Minimum balance threshold",
```

Comment:
If we want the dev to provide wei, we should explicitly make that clear and remind them the token's decimals matter. E.g. 0.1 ETH and 0.1 USDC will have different values.

I'm debating if tokens ("0.1") is more intuitive and less error prone (to get decimals wrong). Also precision doesn't matter because it's just a threshold anyway.